### PR TITLE
Add jest testing

### PR DESCRIPTION
**Pull request by jkozol**
Basic jest tests are added for the landing page. These tests use the react testing library.

### CODE REVIEW

File: src/test/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "env": {
+      "jest": true,
+      "es6": true,
+    },
+    "rules": {
+      "react/display-name": "off",
+      "react/prop-types": "off"
+    }
+  }
```
Comment (Gundersanne) [inline src/test/.eslintrc:10]: Nitpick, but newline at the end of file. It results in weird diffs on github :p

File: .travis.yml [hunk 2]
```diff
@@ -16,4 +20,4 @@ env:
   - REPO="git@github.com:osbuild/image-builder-frontend-build"
   - REPO_DIR="image-builder-frontend-build"
   - BRANCH=${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}
-  - NODE_OPTIONS="--max-old-space-size=4096 --max_old_space_size=4096"
+  - NODE_OPTIONS="--max-old-space-size=4096 --max_old_space_size=4096"
```
Comment (Gundersanne) [inline .travis.yml:23]: ditto about the newline

File: .travis.yml [hunk 3]
```diff
@@ -4,10 +4,14 @@ notifications:
...
 node_js:
   - '10'
+cache:
+  directories:
+  - node_modules
 install:
   - npm ci
 script:
   - npm run travis:verify
+  - npm test
```
Comment (Gundersanne) [inline .travis.yml:14]: This is not necessary, `"travis:verify"` runs `"travis:build lint test"`

**Review by Gundersanne — CHANGES_REQUESTED** [review]
Very nice, thank you! 2 little nitpicks

File: .travis.yml (hunk 2 above)
Comment (jkozol) [reply .travis.yml]: Added.

File: src/test/.eslintrc (hunk 1 above)
Comment (jkozol) [reply src/test/.eslintrc]: I have fixed the end of the file.

**Comment by jkozol** [comment]
@Gundersanne Thanks for the review. I made your requested updates.

**Pull request merged**